mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -370,6 +379,44 @@ func httpDataSchema() *schema.Resource {
 	}
 }
 
+func azureBlobStorageDataSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"storage_account": {
+				Required:    true,
+				Type:        schema.TypeString,
+				Description: `The name of the Azure Storage account.`,
+			},
+			"container": {
+				Required:    true,
+				Type:        schema.TypeString,
+				Description: `The container to transfer from the Azure Storage account.`,
+			},
+			"path": {
+				Required:    true,
```

Comment:
Can it be required and still be an empty string?

Reply:
No, changed it to Optional + Computed